### PR TITLE
Update configuration.rst

### PR DESCRIPTION
**Pull request by MrBuddyCasino**
Fixed description of CONNECT_BOOTSTRAP_SERVERS property.

### CODE REVIEW

**Comment by MrBuddyCasino** [comment]
[clabot:check]

**Comment by MrBuddyCasino** [comment]
[clabot:check]

**Comment by gAmUssA** [comment]
@joel-hamill kindly review

File: docs/configuration.rst [hunk 1]
```diff
@@ -295,7 +295,7 @@ The following settings must be passed to run the Kafka Connect Docker image.
 
 ``CONNECT_BOOTSTRAP_SERVERS``
 
-  A unique string that identifies the Connect cluster group this worker belongs to.
+  A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. This list should be in the form ``host1:port1,host2:port2,...``.
```
Comment (joel-hamill) [inline docs/configuration.rst]: suggest 
```
This list must use the following format: ``host1:port1,host2:port2,...``
```

**Review by joel-hamill — APPROVED** [review]
One grammar suggestion

**Pull request closed without merging**